### DISCUSSION TITLE
Import Skripte mit jarvis.0, v3.2.0-rc.5

### DISCUSSION THREAD

**Discussion by tschortschde** [mcuiobroker/docuJarvis#57] [Q&A] [answered]
Hallo,
ich nutze nun jarvis.0, v3.2.0-rc.5 und möchte weiterhin auf meine Skripts mit der import-Funktion darauf zugreifen. 
Leider funktioniert das nicht mehr im iobroker

Folgende Hilfe habe ich mir bereits angesehen:
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis-v3/besonderheiten-v3/scripts-3.2.x-pro#iobroker-files-dateien-verzeichnis-benutzerdaten-jarvis-scripts-nutzen

![image](https://github.com/user-attachments/assets/b343805f-9dce-489b-8371-7544e73aed39)

Wie muss ich vorgehen?

VG

**Comment by mcuiobroker** [comment]
Cache gelöscht?
Zeig mal in Dateien die jarvis-scripts.

**Reply by tschortschde** [reply]
Ja, Cache habe ich gelöscht
In der Konsole habe ich folgenden Fehler:
![image](https://github.com/user-attachments/assets/b6fce75b-b4c4-454f-928a-667c454e2881)

Hier die Dateien in jarvis-scripts:
![image](https://github.com/user-attachments/assets/e74060c4-fa62-4d11-ad58-0beec3b98203)

Hier die Datei:
[0_userdata.0_jarvis-scripts_widgetMinimize.txt](https://github.com/user-attachments/files/18201821/0_userdata.0_jarvis-scripts_widgetMinimize.txt)
(Die Datei habe ich hier zu txt umbenannt, um sie ins Forum hochladen zu können...)

Wenn ich das Skript direkt in den Bereich "Skripts" einbinde, funktioniert es !?

**Reply by mcuiobroker** [reply]
Versuch mal die rc.12
Jarvis Instanz neu gestartet? 
Es sieht sonst alles korrekt aus.

**Comment by tschortschde** [comment]
Auch unter der rc.12 funktioniert es leider nicht.
Ich bekomme in den Protokollen vom iobroker folgende Fehlermeldung:
![image](https://github.com/user-attachments/assets/393d7446-2c34-4283-ab3a-341b81c8a423)

**Reply by tschortschde** [reply, marked as answer]
Jetzt habe ich es geschafft 
@import "http://192.168.178.145:8082/0_userdata.0/jarvis-scripts/widgetMinimize.js";

**Reply by mcuiobroker** [reply]
Port 8400 muss aber auch funktionieren
